src/formio/components/AddressNL.scss

```diff
@@ -0,0 +1,19 @@
+@import '~microscope-sass/lib/responsive';
```

Comment:
```suggestion
@use 'microscope-sass/lib/bem';

// @import instead of @use because breakpoints are defined globally
@import 'microscope-sass/lib/responsive';
```